src/SerialDataCollector.cpp

```diff
@@ -957,6 +973,45 @@ namespace splash
         dataset.close();
     }
 
+    CollectionType* SerialDataCollector::readDataSetMeta(H5Handle h5File,
+            int32_t id,
+            const char* name,
+            const Dimensions dstBuffer,
+            const Dimensions dstOffset,
+            const Dimensions srcOffset,
+            Dimensions &sizeRead,
+            uint32_t& srcDims)
+    throw (DCException)
+    {
+        log_msg(2, "readDataSetMeta");
+
```

Comment:
that's a private member such as `readDataSet` and state of the `SerialDataCollector` (open, readable, ...) has already been checked at this point